app/src/main/java/chat/rocket/android/main/ui/MainActivity.kt

```diff
@@ -353,99 +351,127 @@ class MainActivity : AppCompatActivity(), MainView, HasActivityInjector,
         progressDialog = null
     }
 
+    // WIDECHAT
+    private fun requestLocationPermissions() {
```

Comment:
Starting with Android 8.1 (API 27), apps must be granted the ACCESS_COARSE_LOCATION (or ACCESS_FINE_LOCATION) permission in order to obtain results from WifiInfo.getSSID() or WifiInfo.getBSSID().

What we can do is only ask for location permission if android version is 8.1 or higher
using 

```
if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O_MR1) {
        requestLocationPermissions()
}
```